cypress-tests/cypress/e2e/configs/Payment/WorldPay.js

```diff
@@ -45,6 +45,14 @@ const successfulThreeDsTestCardDetailsRequest = {
   card_cvc: "737",
 };
 
+const failedNoThreeDsCardDetails = {
```

Comment:
also, can you please post a screenshot of tests running against worldpay in the pr description?